migration/sql-files/003-profiles.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TYPE profiles AS ENUM ('free', 'paid');
```

Comment:
You need to register the migration here https://github.com/fabric8-services/fabric8-tenant/blob/master/migration/migration.go#L105